specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -60,47 +59,45 @@ image:Platform_Spec-10.png[image]
 The EJB, servlet, application client, and
 Connector specifications include the XML Schema definition of the
 associated module level deployment descriptors and component packaging
-architecture required to produce Java EE modules. (The application
+architecture required to produce Jakarta EE modules. (The application
 client specification is found in
 link:#a3293[See CHApter] of this
 document.)
 
-A Java EE module is a collection of one or more
-Java EE components (web, EJB, application client, or Connector) with an
+A Jakarta EE module is a collection of one or more
+Jakarta EE components (web, EJB, application client, or Connector) with an
 optional module deployment descriptor of that type. Any number of
 components of the same container type can be packaged together with a
-single Java EE deployment descriptor appropriate to that container type
-to produce a Java EE module. Components of different container types may
-not be mixed in a single Java EE module, except for the packaging of EJB
+single Jakarta EE deployment descriptor appropriate to that container type
+to produce a Jakarta EE module. Components of different container types may
+not be mixed in a single Jakarta EE module, except for the packaging of EJB
 components within a web module.
 
-* A Java EE module represents the basic unit of
-composition of a Java EE application. In some cases a single Java EE
-module (not necessarily packaged into a Java EE application package)
+* A Jakarta EE module represents the basic unit of
+composition of a Jakarta EE application. In some cases a single Jakarta EE
+module (not necessarily packaged into a Jakarta EE application package)
 will contain an entire application. In other cases an application will
-be composed of multiple Java EE modules.
-* The deployment descriptor for a Java EE
+be composed of multiple Jakarta EE modules.
+* The deployment descriptor for a Jakarta EE
 module contains declarative data required to deploy the components in
-the module. The deployment descriptor for a Java EE module also contains
+the module. The deployment descriptor for a Jakarta EE module also contains
 assembly instructions that describe how the components are composed into
 an application.
-* Starting with version 5 of the Java EE
-platform, a web application module, an enterprise bean module, or an
+* Starting with version 5 of the Jakarta EE, a web application module, an enterprise bean module, or an
```

Comment:
Question here as this talks about version 5 maybe this should remain as Java EE

Reply:
agreed. reverted.